README.md

```diff
@@ -61,72 +64,111 @@ Available versions:
   ...
 ```
 
-Lets install a new version of Python (3.9.2). We can do this using the
- following command
+Lets install a new version of Python (3.11). We can do this using the following
+command
 
-```bash
-pyenv install 3.9.2
+```console
+$ pyenv install 3.11
+```
 
+To set the global Python version to the new version 3.11:
+
+```console
+$ pyenv global 3.11
 ```
 
-To set the global Python version to the one we have installed
+Confirm the current global Python version (full version 3.11.x may differ):
 
-```bash
-pyenv global 3.9.2
+```console
+$ python3 --version
+Python 3.11.4
 ```
 
-Now we can run Python code with the version of Python we want to use.
+Now we can run Python code with version 3.11.x.
 
-***
+Let's set the global Python version back to 3.8.13:
+
+```console
+$ pyenv global 3.8.13
+```
+
+Confirm the global Python version:
+
+```console
+$ python3 --version
+Python 3.8.13
+```
+
+---
 
 ## Pipenv
 
 Pyenv allows us to manage the version of Python we are working with.
 
 What if we want to manage dependencies as well?
 
-Pipenv  automatically creates and manages a virtualenv for your projects,
- it also adds/removes packages from your Pipfile as you install/uninstall packages
+Pipenv automatically creates and manages a virtualenv for your projects, it also
+adds/removes packages from your Pipfile as you install/uninstall packages
 
-To install pipenv follow the instructions here [pipenv install](https://pipenv.pypa.io/en/latest/installation/)
+To install pipenv follow the instructions here
+[pipenv install](https://pipenv.pypa.io/en/latest/installation/)
 
 To create a virtual environment using pipenv we can use the following command.
 
 ```bash
-pipenv --python 3.8.13
+pipenv --python 3.11
 ```
 
 In the Python flag we can define which version of `--python` we want to use.
```

Comment:
Does this mean that this program will use python 3.11 even when our global version is 3.8.13? Does that require any extra setup, or will we be able to run the program as long as we have 3.11 installed on our machine? If so, we might want to say so explicitly - as is, it might still be confusing to students as to why we installed 3.11, switched to it globally, then switched back to 3.8.13, then decided to use 3.11 for this program.

Could include something like:

> Since we've set the python version to 3.11, this program will run using python 3.11, even though our global version is 3.8.13. As long as we have a specific python version installed on our computer, we can use it locally in specific programs without having to reset our global python version.

But only if what I've written is an accurate representation of the truth.

Reply:
added explanation and updated sample Python program to print version.